cypress/support/e2e.ts

```diff
@@ -115,6 +116,10 @@ beforeEach(() => {
         })
       )
     })
+    // Mock our own token list responses to avoid the latency of IPFS.
+    .intercept('https://gateway.ipfs.io/ipns/tokens.uniswap.org', TokenListJSON)
+    .intercept('https://gateway.ipfs.io/ipns/extendedtokens.uniswap.org', { statusCode: 201, body: { tokens: [] } })
+    .intercept('https://gateway.ipfs.io/ipns/unsupportedtokens.uniswap.org', { statusCode: 201, body: { tokens: [] } })
```

Comment:
can these be overridden in individual tests? there may be a use case for that especially the unsupported list